ui_framework/components/table/table_row_check_box_cell.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KuiTableRowCell } from './table_row_cell';
+import { KuiTableRowCellLiner } from './table_row_cell_liner';
+
+export const KuiTableRowCheckBoxCell = ({ onChange, isChecked, className, ...rest }) => {
+  const classes = classNames('kuiTableRowCell--checkBox', className);
+  return <KuiTableRowCell className={ classes } {...rest} >
+    <KuiTableRowCellLiner>
+      <input type="checkbox" className="kuiCheckBox" onChange={ onChange } checked={ isChecked } />
```

Comment:
Do we have a style rule for this kind of thing?  It'd be easier to know what the rule is so I can get it right the first time, because I generally prefer fewer lines if it fits within the per line char limit which I believe is 120. Whether the rule is something like  "if there are x number or greater props, put them individual lines",  or "if your props go over x characters, put them on individual lines".  Knowing this will save both of us time, especially as more developers start creating react code.  Unless there is a hard rule, I think it will be impossible to keep up with these kind of review comments.